src/System.Device.Gpio/System/Device/Gpio/Drivers/LibGpiodDriver.Linux.cs

```diff
@@ -38,15 +38,15 @@ public LibGpiodDriver(int gpioChip = 0)
 
         protected internal override void AddCallbackForPinValueChangedEvent(int pinNumber, PinEventTypes eventTypes, PinChangeEventHandler callback)
         {
-            if (eventTypes.HasFlag(PinEventTypes.Rising) || eventTypes.HasFlag(PinEventTypes.Falling))
+            if ((eventTypes & PinEventTypes.Rising) != 0 || (eventTypes & PinEventTypes.Falling) != 0)
```

Comment:
I don't think we should change this - instead we should make HasFlag faster - HasFlag is much cleaner than this

Reply:
This was a suggestion by @danmosemsft on PR #133 since just doing a bitwise operation would be faster. I know that HasFlag is a bit cleaner since it validates that the enum types specified are the same, but given that we control all the places where we do these checks I believe that we should be fine to just do the bitwise operation to have a better perf.

Reply:
I feel like this is a bit microptimization since we don't expect people to register events too frequently (expecting once per runtime of the app?)

Reply:
Agreed but that doesn't mean we shouldn't optimize where we can, specially if it is as easily as it is here where we aren't increasing complexity in the code.